simbad/mr/phaser_mr.py

```diff
@@ -283,20 +297,28 @@ def run(self):
             i.setJOBS(1)
             i.setREFL_DATA(r.getREFL_DATA())
             i.setROOT("phaser_mr_output")
-            i.addENSE_PDB_RMS("PDB", pdbin, 0.6)
+            i.addENSE_PDB_RMS("PDB", pdbin, 1.2)
             i.setCOMP_BY("SOLVENT")
             i.setCOMP_PERC(self.solvent)
             i.addSEAR_ENSE_NUM('PDB', self.nmol)
+            if self.sgalternative == "all":
```

Comment:
Maybe slightly neater and more future proof would be to use a dictionary to keep all those equivalents. You could then populate your `argparse` choices dynamically.

```python
SGALTERNATIVES = {
    'all': 'ALL',
    'enant': 'HAND',
}
```
Then define the flag in Phaser using the following bit
```python
i.setSGAL_SELE(SGALTERNATIVES.get(self.sgalternative, 'NONE'))
```
and do your `argparse` business with
```python
from simbad.mr.phaser_mr import SGALTERNATIVES
... choices=SGALTERNATIVES.keys() ...
```

Reply:
Double-check there are no circular dependencies but this way you can add a value into the dictionary for a new mode without changing anything else. If the user provides nothing, it will default to 'NONE' automatically